workers/unity/Packages/com.improbable.gdk.core/Systems/CommandSystem.cs

```diff
@@ -11,6 +11,10 @@ namespace Improbable.Gdk.Core
     public class CommandSystem : ComponentSystem
     {
         private readonly List<ICommandManager> managers = new List<ICommandManager>();
+
+        private readonly Dictionary<uint, Dictionary<uint, ICommandManager>> componentIdToCommandIdToManager =
```

Comment:
is this needed?

Reply:
No, it was replaced and I forgot to remove it.